HW_05/class_Circle.py

```diff
@@ -0,0 +1,24 @@
+from typing import Union
+
+from HW_05.class_Point import Point
+
+
+class Circle:
+    def __init__(self, x: Union[int, float], y: Union[int, float], radius: Union[int, float]):
```

Comment:
If you are using new python this annotation is deprecated: 
def __init__(self, x: int | float, y: int | float):